contracts/characters.sol

```diff
@@ -125,16 +125,20 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
 
     Garrison public garrison;
 
+    mapping(uint256 => uint256) public bonusPower;
```

Comment:
I'd really like for us to use the nftVars mapping for things like this.
In the long run, it will save on setters and getters, as well as allow us to place logic outside of the contract (like into the burning manager. We could even move soulSupply there.
Even the nft bridge will have an easier time grabbing bulk nftVars in an array rather than calling multiple getters.